src/main/java/fr/openent/viescolaire/utils/DateHelper.java

```diff
@@ -28,7 +28,18 @@ public class DateHelper {
     private static final String  END_DATE = "endDate";
     private static final String  DAY_OF_WEEK = "dayOfWeek";
 
+    public static Date parseDate(String dateString, String format) {
+        Date date = new Date();
 
+        SimpleDateFormat sdf = new SimpleDateFormat(format);
+        try {
+            date = sdf.parse(dateString);
+        } catch (ParseException e) {
+            LOGGER.error("[Viescolaire@DateHelper::parseDate] Error when casting date: ", e);
```

Comment:
```suggestion
            LOGGER.error(String.format("[Viescolaire@%s::parseDate] Error when casting date: %s", this.getClass.getSimpleName(), e.getMessage());
```

Meme chose pour les deux fonction en dessus